bode/bode/resources/tasks/schemas.py

```diff
@@ -23,3 +23,4 @@ class TaskSchema(BaseSchema):
     due_date = fields.DateTime()
     status = fields.String()
     tags = fields.List(fields.Nested(TagSchema))
+    relation_types = fields.List(fields.String())
```

Comment:
I think it may be useful if we return a list of objects containing: `{ taskId: string, relation_type: RelationType }`. Then we could link directly to this task, display a tooltip or sth like this